inc/sitemaps/class-sitemaps-cache.php

```diff
@@ -174,7 +411,11 @@ public static function clear( $types = array() ) {
 	public static function register_clear_on_option_update( $option, $type = '' ) {
 
 		self::$cache_clear[ $option ] = $type;
-		add_action( 'update_option', array( __CLASS__, 'clear_on_option_update' ) );
+
+		$function = array( __CLASS__, 'clear_on_option_update' );
+		if ( false === has_action( 'update_option', $function ) ) {
+			add_action( 'update_option', $function );
```

Comment:
This can be moved to a place where it always happens, instead of with a conditional.